### DISCUSSION TITLE
Portrait Mode

### DISCUSSION THREAD

**Discussion by jeffboy70** [KillerInk/FreeDcam#175] [Q&A] [answered]
Is there a way to change FreeDCam to portrait mode?  I looked through the settings and didn't see an option but I probably just missed it.

**Comment by KillerInk** [comment, marked as answer]
there is no explicit setting. normaly the image orientation is the same as you hold your device.
but you can force it with locking device orientation in system settings

**Reply by jeffboy70** [reply]
It doesn't change to portrait automatically, and it doesn't change when you lock portrait mode either.

**Reply by KillerInk** [reply]
new version fix it. thx
